front/public/locales/en/errors.json

```
@@ -139,6 +139,7 @@
       "ExceptionNotFound": "Exception '{{exception_key}}' could not be found",
       "InfraNotFound": "Infrastructure '{{infra_id}}' could not be found",
       "NotFound": "Paced train '{{paced_train_id}}' could not be found",
+      "OperationalPointNotFound": "Operational Point '{{operational_point_id}}' could not be found",
```

```suggestion
      "OperationalPointNotFound": "Operational point '{{operational_point_id}}' could not be found",
```
But I saw that “Operational Point” was also written in the same file, so it looks good to me

---

Fixed in https://github.com/OpenRailAssociation/osrd/compare/1b68ef5c674315c9c5cb37df2ad00bf8d0b37ed5..1e2a057774eca6d29e2e9cb589399e13a9f5645d.